addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiDeviceBaseHandler.java

```diff
@@ -74,7 +77,12 @@ public XiaomiDeviceBaseHandler(Thing thing) {
     @Override
     public void initialize() {
         setItemId((String) getConfig().get(ITEM_ID));
-        updateThingStatus();
+        onlineCheckTask = scheduler.scheduleWithFixedDelay(new Runnable() {
```

Comment:
Next time I would prefer a this:: updateThingStatus instead of the Runnable:

```java
scheduler.scheduleWithFixedDelay(this:: updateThingStatus, 0, ONLINE_TIMEOUT_MILLIS / 2, TimeUnit.MILLISECONDS);
```